doc/ChangeLog

```diff
@@ -1,4 +1,122 @@
 ===============================================================
+Tag name: ctsm5.1.dev154
+Originator(s): slevis (Samuel Levis,UCAR/TSS,303-665-1310)
+Date: Wed Nov 22 09:53:01 MST 2023
+One-line Summary: New params files: Changes for Meier roughness, MIMICS, and SNICAR, and changes to leafcn and k*_nonmyc
+
+Purpose and description of changes
+----------------------------------
+
+This PR (#2258) addresses several issues:
+1) Start using existing new params file for Meier roughness:
+/glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/paramdata/ctsm51_params.RMz0.c231011.nc
+and include bug-fix #2219.
+2) Update forcing heights per #2071.
+3) Update params file for MIMICS per #1845.
+4) Make leafcn for pfts 15 and 16 the same per #2184.
+5) Switch the values of params kc_nonmyc and kn_nonmyc per #2120.
+6) Move SNICAR parameters to ctsm51, clm50, and clm45 params files per #2247.
+
+See #2258 and the above issues for a list of contributors.
+
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[x] clm5_1
+
+[x] clm5_0
+
+[x] ctsm5_0-nwp
+
+[x] clm4_5
+
+
+Bugs fixed or introduced
+------------------------
+CTSM issues fixed (include CTSM Issue #):
+Fixes #2219
+Fixes #2071
+Fixes #1845
+Fixes #2184
+Fixes #2120
+Fixes #2247
+
+
+Notes of particular relevance for users
+---------------------------------------
+Changes to CTSM's user interface (e.g., new/renamed XML or namelist variables):
+ SNICAR namelist variable fresh_snw_rds_max moved to the params file.
+
+Changes made to namelist defaults (e.g., changed parameter values):
+ SNICAR namelist variable fresh_snw_rds_max moved to the params file.
+ Pointing to new params files for clm4_5, clm5_0, clm5_1.
+
+Changes to the datasets (e.g., parameter, surface or initial files):
+ New clm5_1 params file with new parameters and with modified values of existing parameters.
+ New clm5_0 and clm4_5 params files with new parameters for SNICAR.
+ ./rimport on the new params files fails with "No space left on device" but the 4 files are safe here:
+ /glade/u/home/slevis/paramfiles/*_params.c231117.nc
```

Comment:
Final comment before I merge:
./rimport on the new params files fails with "No space left on device"
The files are safe in my home directory: /glade/u/home/slevis/paramfiles.

Reply:
**Update**  I have now completed this pending item:
```
./rimport -file lnd/clm2/paramdata/clm45_params.c231117.nc
./rimport -file lnd/clm2/paramdata/clm50_params.c231117.nc
./rimport -file lnd/clm2/paramdata/ctsm51_params.c231117.nc
./rimport -file lnd/clm2/paramdata/ctsm51_ciso_cwd_hr_params.c231117.nc
```